From 02473fb3463ac557fc2f8939a303315ece5a2094 Mon Sep 17 00:00:00 2001
From: novenary <novenary@kwak.zip>
Date: Sun, 18 May 2025 02:07:54 +0300
Subject: [PATCH 6/6] Quest book: fix quantum coolant stage reset commands

This fixes quest descriptions always jumping to the top in HM/EM.
---
 config/ftbquests/quests/chapters/dependency_chain.snbt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/ftbquests/quests/chapters/dependency_chain.snbt b/config/ftbquests/quests/chapters/dependency_chain.snbt
index 04adbc92..deca6ef3 100644
--- a/config/ftbquests/quests/chapters/dependency_chain.snbt
+++ b/config/ftbquests/quests/chapters/dependency_chain.snbt
@@ -525,7 +525,7 @@
 			invisible: true
 			rewards: [{
 				auto: "invisible"
-				command: "/ftbquests change_progress @p reset 776886E9296912DD"
+				command: "/ftbquests change_progress @p reset 0412D46926C075FF"
 				elevate_perms: true
 				id: "3EB61E55554FA706"
 				silent: true
@@ -544,7 +544,7 @@
 			invisible: true
 			rewards: [{
 				auto: "invisible"
-				command: "/ftbquests change_progress @p reset 10E63A0428E6BA25"
+				command: "/ftbquests change_progress @p reset 776886E9296912DD"
 				elevate_perms: true
 				id: "4CCF413223F39D3F"
 				silent: true
-- 
2.47.2

